src/schema/v2/me/myCollectionAuctionResults.ts

```diff
@@ -12,49 +12,49 @@ import { pageable } from "relay-cursor-paging"
 import { createPageCursors } from "schema/v2/fields/pagination"
 import { ResolverContext } from "types/graphql"
 import ArtworkSizes from "../artwork/artworkSizes"
-import { auctionResultConnection, AuctionResultSorts } from "../auction_result"
+import {
+  auctionResultConnection,
+  AuctionResultSorts,
+  AuctionResultsState,
+} from "../auction_result"
 import { MAX_ARTISTS } from "./myCollectionInfo"
 
 const MyCollectionAuctionResults: GraphQLFieldConfig<any, ResolverContext> = {
   type: auctionResultConnection.connectionType,
   description: "A list of auction results from artists in My Collection",
   args: pageable({
-    sort: AuctionResultSorts,
-    organizations: {
-      type: new GraphQLList(GraphQLString),
-      description: "Filter auction results by organizations",
-    },
-    sizes: {
-      type: new GraphQLList(ArtworkSizes),
-      description: "Filter auction results by Artwork sizes",
+    allowEmptyCreatedDates: {
+      type: GraphQLBoolean,
+      defaultValue: true,
+      description:
+        "Filter auction results by empty artwork created date values",
```

Comment:
**suggestion:** Not quite sure if the description is matching the param name and describing what the param does. (allow vs. filtering).

Reply:
this was there for a while though, I only updated the order of the args